ShapekeyMaster/GUI/UI.cs

```diff
@@ -205,7 +239,15 @@ private static void GuiWindowControls(int windowId)
 
 				DisplayHeaderMenu();
 
-				ShapeKeyMaster.SimpleMode.Value = GUILayout.Toggle(ShapeKeyMaster.SimpleMode.Value, ShapeKeyMaster.CurrentLanguage["simple"]);
+				GUILayout.BeginHorizontal();
+
+				ShapeKeyMaster.SimpleMode.Value = GUILayout.Toggle(ShapeKeyMaster.SimpleMode.Value, ShapeKeyMaster.CurrentLanguage["simple"], UIUserOverrides.getToggleStyleOverride());
+				if (ShapeKeyMaster.SimpleMode.Value)
+				{
+					ShapeKeyMaster.SimpleMode_ShowMoreFuntions.Value = GUILayout.Toggle(ShapeKeyMaster.SimpleMode_ShowMoreFuntions.Value, ShapeKeyMaster.CurrentLanguage["showMoreFuntions"], UIUserOverrides.getToggleStyleOverride());
```

Comment:
Let's try to keep it clean.

```suggestion
					ShapeKeyMaster.SimpleMode_ShowMoreFunctions.Value = GUILayout.Toggle(ShapeKeyMaster.SimpleMode_ShowMoreFunctions.Value, ShapeKeyMaster.CurrentLanguage["showMoreFunctions"], UIUserOverrides.getToggleStyleOverride());
```

Reply:
Typo fixed.